docs/platform/toc_all.md

```diff
@@ -1,7 +1,19 @@
-# Overview
-## [Development Workflow](/platform/about/work-flow.md)
-## [Git Repository Structure](/platform/about/git-repo-structure.md)
-## [Typographic Conventions](/platform/about/conventions.md)
+# What is Tizen?
```

Comment:
> Just for the reference.
> I ignored almost 66 files as same fix is already tried on #844.
> It looks like file updates for #844 are included here also by mistake. 👎 

It's not a mistake. It was intended because the menu tree is based on PR 844. 
I divided PRs to get a review from TPL.
If PR #844 is merged, the PR will be gracefully merged without conflict.